plugins/org.jkiss.dbeaver.ui.editors.sql/src/org/jkiss/dbeaver/ui/editors/sql/SQLEditor.java

```diff
@@ -3644,10 +3644,15 @@ public DBDDataReceiver getDataReceiver(final SQLQuery statement, final int resul
                 String tabName = null;
                 String queryText = CommonUtils.truncateString(statement.getText(), 1000);
                 DBPDataSourceContainer dataSourceContainer = getDataSourceContainer();
+                
+                String notAvaliable = SQLEditorMessages.sql_editor_not_avaliable;
+                String dataSourceContainerName = dataSourceContainer == null ? notAvaliable : dataSourceContainer.getName();
+                String processedQueryText = CommonUtils.isEmpty(queryText) ? notAvaliable : queryText;
+                
                 String toolTip =
-                    "Connection: " + (dataSourceContainer == null ? "N/A" : dataSourceContainer.getName()) + GeneralUtils.getDefaultLineSeparator() +
-                    "Time: " + new SimpleDateFormat(DBConstants.DEFAULT_TIMESTAMP_FORMAT).format(new Date()) + GeneralUtils.getDefaultLineSeparator() +
-                    "Query: " + (CommonUtils.isEmpty(queryText) ? "N/A" : queryText);
+                    NLS.bind(SQLEditorMessages.sql_editor_data_receiver_result_name_tooltip_connection,dataSourceContainerName ) + GeneralUtils.getDefaultLineSeparator() +
```

Comment:
- Line is longer than 140 characters (found 171).
- , is not followed by whitespace
-  ) is preceded with whitespace.
You need to fix this also